scripts/exregional_run_fcst.sh

```diff
@@ -756,6 +755,49 @@ fi
 #
 #-----------------------------------------------------------------------
 #
+# Call the function for updating the &time section of namelist.fire
+#
+#-----------------------------------------------------------------------
+#
+if [ $(boolify "${UFS_FIRE}") = "TRUE" ]; then
+  FCST_END_DATE=$( $DATE_UTIL --utc --date "${PDY} ${cyc} UTC + ${FCST_LEN_HRS} hours" "+%Y%m%d%H%M%S" )
+  # This horrible syntax $((10#$VARNAME)) is to force bash to treat numbers as decimal instead of
+  # trying to octal all up in our business
+  settings="
+  &time
+    start_year   = $((10#${CDATE:0:4})),
+    start_month  = $((10#${CDATE:4:2})),
+    start_day    = $((10#${CDATE:6:2})),
+    start_hour   = $((10#${CDATE:8:2})),
+    start_minute = 00,
+    start_second = 00,
+    end_year     = $((10#${FCST_END_DATE:0:4})),
+    end_month    = $((10#${FCST_END_DATE:4:2})),
+    end_day      = $((10#${FCST_END_DATE:6:2})),
+    end_hour     = $((10#${FCST_END_DATE:8:2})),
+    end_minute   = $((10#${FCST_END_DATE:10:2})),
+    end_second   = $((10#${FCST_END_DATE:12:2})),
+  /
+"
+
+  echo $settings | uw config realize --update-format nml --input-format nml --output-format nml --input-file "${FIRE_NML_FP}" -o "${FIRE_NML_FN}"
```

Comment:
Here is where you might change to `uw template render` suggested above, but use the `-o "${FIRE_NML_FN}"` to have it go to the appropriate file instead of stdout.

This would require a change to the error handling below, too.